test/system/001-basic.bats

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+#
+# Simplest set of podman tests. If any of these fail, we have serious problems.
+#
+
+load helpers
+
+# Override standard setup! We don't yet trust podman-images or podman-rm
```

Comment:
Does "standard" mean "Defined in another file"?  Might not hurt to simply add that reference.

Reply:
It's right under `load helpers`; I have to assume that the reader has some familiarity with BATS conventions, or enough familiarity with shell scripting to know that `load` is not a shell function and must have something to do with setup. I'm really not sure how to spell it out more without making it unhelpfully verbose.

Reply:
Oh that's true.  Yeah my eyes often skip over the important details right in-front of them :grin: To be fair to myself...I did have one sip remaining in my coffee mug.